source/form.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body class="page">
+  <div class=page-wrapper>
+    <header class="page-header">
+      <nav class="main-nav">
+        <a class="page-header__logo" href="index.html">
+          <img src="logo.svg" width="102" height="84" alt="Логотип City of Sedona">
+        </a>
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item--active">
+            <a href="index.html">
+              Главная
+            </a>
+          </li>
+          <li class="main-nav__item">
+            <a href="catalog.html">
+              Фото и Видео
+            </a>
+          </li>
+          <li class="main-nav__item">
+            <a>
+              Форма отзыва
+            </a>
+          </li>
+          <li class="main-nav__item">
+            <a href="https://htmlacademy.ru">
+              Html Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page-main">
+      <h1 class="page-main__header">
+        Оставьте свой озыв
+      </h1>
+      <img src="intro.jpeg" width="102" height="84" alt="Фото города">
+      <p class="page-main__text">
+        Помогите нашим отелям стать лучше! оставьте отзыв о них, а также
+        о посещенных вами достопримечательностях
+      </p>
+      <form class="form" action="https://echo.htmlacademy.ru" method="POST">
+        <fieldset class="form__item">
+          <legend>Представьтесь:</legend>
+          <label class="form__label">
+            Имя*:
+            <input class="form__input" type="text" name="name" placeholder="Петр" required>
+          </label>
+          <label class="form__label">
+            Фамилия*:
+            <input class="form__input" type="text" name="surname" placeholder="Иванов" required>
+          </label>
+          <label class="form__label">
+            Отчество:
+            <input class="form__input" type="text" name="patronymic" placeholder="Александрович">
+          </label>
+        </fieldset>
+
+        <fieldset class="form__item">
+          <legend>Контактная информация:</legend>
+          <label class="form__label">
+            Контактный телефон*:
+            <input class="form__input" type="text" name="tel" placeholder="Введите телефон" required>
+          </label>
+          <label class="form__label">
+            Элетронная почта*:
+            <input class="form__input" type="text" name="e-mail" placeholder="Введите e-mail" required>
```

Comment:
```suggestion
            <input class="form__input" type="email" name="e-mail" placeholder="Введите e-mail" required>
```
Описано в [этом пункте](https://up.htmlacademy.ru/adaptive/23/project/sedona#sedona-5-2) ТЗ.